eng/versioning/version_data.txt

```diff
@@ -43,3 +43,5 @@ com.microsoft.azure:azure-media;1.0.0-beta.1;1.0.0-beta.1
 com.microsoft.azure:azure-spring-boot;2.2.4;2.2.5-beta.1
 com.microsoft.azure:azure-spring-boot-starter;2.2.4;2.2.5-beta.1
 com.microsoft.azure:azure-active-directory-spring-boot-starter;2.2.4;2.2.5-beta.1
+com.microsoft.azure:azure-servicebus-jms-spring-boot-starter;2.2.4;2.2.5-beta.1
```

Comment:
Shouldn't these be being added to version_client.txt instead of version_data.txt like the other spring libraries are being added?

Reply:
Moved.